src/SprykerEco/Zed/Unzer/Business/Import/UnzerPaymentMethodImporter.php

```diff
@@ -88,4 +109,63 @@ public function performPaymentMethodsImport(UnzerKeypairTransfer $unzerKeypairTr
 
         $this->paymentImportAdapter->createPaymentMethods($paymentProviderTransfer, $paymentMethodTransfers);
     }
+
+    /**
+     * @param \Generated\Shared\Transfer\PaymentMethodsTransfer $paymentMethodsTransfer
+     * @param \Generated\Shared\Transfer\UnzerCredentialsTransfer $unzerCredentialsTransfer
+     *
+     * @return \Generated\Shared\Transfer\PaymentMethodsTransfer
+     */
+    protected function appendChildPaymentMethods(
+        PaymentMethodsTransfer $paymentMethodsTransfer,
+        UnzerCredentialsTransfer $unzerCredentialsTransfer
+    ): PaymentMethodsTransfer {
+        if (!$unzerCredentialsTransfer->getUnzerKeypair()) {
+            return $paymentMethodsTransfer;
+        }
+
+        $childPaymentMethodsTransfer = $this->unzerPaymentMethodsAdapter->getPaymentMethods($unzerCredentialsTransfer->getUnzerKeypair());
+        $unzerPaymentMethodKeys = $this->extractPaymentMethodKeysFromPaymentMethodsTransfer($paymentMethodsTransfer);
+
+        foreach ($childPaymentMethodsTransfer->getMethods() as $paymentMethodTransfer) {
+            if (!in_array($paymentMethodTransfer->getPaymentMethodKeyOrFail(), $unzerPaymentMethodKeys, true)) {
+                $paymentMethodsTransfer->getMethods()->append($paymentMethodTransfer);
+            }
+        }
+
+        return $paymentMethodsTransfer;
+    }
+
+    /**
+     * @param \Generated\Shared\Transfer\UnzerKeypairTransfer $unzerKeypairTransfer
+     *
+     * @return \Generated\Shared\Transfer\UnzerCredentialsCollectionTransfer|null
+     */
+    protected function getChildUnzerCredentialsCollectionTransfer(
+        UnzerKeypairTransfer $unzerKeypairTransfer
+    ): ?UnzerCredentialsCollectionTransfer {
+        if (!$unzerKeypairTransfer->getIdUnzerCredentials()) {
+            return null;
```

Comment:
I would suggest returning `new UnzerCredentialsCollectionTransfer` and rely on it's `getXXX()->count()`  at usage places